### ISSUE TITLE
Heroku Deploy issues

### ISSUE THREAD

**Issue by Rev3rt4** [dtcarls/ff_bot#18]
I followed the instructions step by step. I created the bot and verified it worked in my groupme chat then deployed the bot using the heroku link. Everything looked to be working after I amended the variables to my fantasy league and bot ID. When I turned it on, I never got an initial message in my groupme chat so I assume it did not link correctly. Any troubleshooting help here would be much appreciated.

**Comment by Rev3rt4** [comment]
I did have to change my league to public before doing the initial process but it was done ahead of time. My league does have a trade deadline of Nov. 21. League is full. The only environment variables I changed was bot_ID and league_ID. I copy-pasted the bot_ID so no way that's incorrect. And again, I made sure the bot worked beforehand. So that leaves league_ID, which I tried typing with spaces and with underscores instead of spaces. I turned it on and still nothing.. Is there a step I'm missing? Does the fact that I'm not getting any init message in my groupme chat mean the bot_ID is the issue?
